assets/stylesheets/test-details.scss

```diff
@@ -414,4 +414,14 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
+    position: relative;
+    left: 0.2em;
+    opacity: 0.8;
```

Comment:
Setting this back to `initial` on hover would likely look good.

Reply:
Done, as far as I understood